Sources/Demo/MapRootController.swift

```diff
@@ -46,10 +46,16 @@ class MapRootController: UIViewController {
     }
 
     func showAnnotations(results: [SearchResult], cameraShouldFollow: Bool = true) {
-        annotationsManager.annotations = results.map {
-            var point = PointAnnotation(coordinate: $0.coordinate)
-            point.textField = $0.name
+        annotationsManager.annotations = results.map { result in
+            var point = PointAnnotation(coordinate: result.coordinate)
+            point.textField = result.name
             UIImage(named: "pin").map { point.image = .init(image: $0, name: "pin") }
+
+            // Present a detail view upon annotation tap
+            point.tapHandler = ((MapContentGestureContext) -> Bool)? { [weak self] _ in
+                self?.present(result: result)
+                return false
```

Comment:
Are you sure you need `false` here?
From the docs:
> Should return `true` if the gesture is handled, or `false` to propagate it to the annotations or layers below.

It looks like now you ignore the returned value in present
```suggestion
                return self?.present(result: result) ?? false
```